public/class-dominant-colors-lazy-loading-public.php

```diff
@@ -282,6 +282,10 @@ public function replace_source_with_dominant_color( $image, $color, $format, $no
 			return $image;
 		}
 
+		if ( preg_match('/class="[^"]*\bdisable-dcll\b[^"]*"/', $image)) { //if the class disable-dcll is present
```

Comment:
Do you need the word boundaries? I think a simpler RegEx would be sufficient, as `disable-dcll` is a very uncommon string that should not lead to many false positives?